pkg/build/apis/build/types.go

```diff
@@ -604,6 +604,10 @@ type BuildSource struct {
 	// Secrets represents a list of secrets and their destinations that will
 	// be used only for the build.
 	Secrets []SecretBuildSource
+
+	// ConfigMaps represents a list of configMaps and their destinations that will
+	// be used only for the build.
+	ConfigMaps []ConfigMapBuildSource
```

Comment:
not sure if @deads2k wanted the internal api changes *with* the update commit, or separate.  I think he wanted it separate (ie only generated code in one of the commits, only manual changes in the other commit).